src/Symfony/Component/Validator/Tests/Constraints/EmailValidatorTest.php

```diff
@@ -34,7 +34,7 @@ public function testLegacyValidatorConstructorStrict()
     {
         $this->validator = new EmailValidator(true);
         $this->validator->initialize($this->context);
-        $this->validator->validate('example@localhost', new Email());
+        $this->validator->validate('example@mywebsite.tld', new Email());
```

Comment:
💡 I had to change these tests because `egulias/email-validator` does not allow `@localhost` domains anymore.

Reply:
This is because the DNSCheckValidation is in use and is considered a reserved top-level (see https://github.com/egulias/EmailValidator/blob/3.x/src/Validation/DNSCheckValidation.php#L50)